src/modules/cache-client/cache-client.service.ts

```diff
@@ -85,16 +86,10 @@ export class CacheClient implements ICacheClient {
     // If refresh token failed or access token is not valid, then sign new identity token
     if (!(await this.isAuthenticated())) {
       getLogger().info('[CACHE CLIENT] obtaining new tokens');
-      delete this._httpClient.defaults.headers.common['Authorization'];
```

Comment:
Is this line still needed? It seems to be unrelated to the authentication protocol change

Reply:
I removed it, because this line do nothing. Next step in this method signs in to ssi-hub, which doesn't need Authorization header and then Authrization header is assigned new value. But of course we can remove this in separate PR